lib/fromRdf.js

```diff
@@ -285,6 +286,10 @@ function _RDFToObject(o, useNativeTypes) {
     if(!type) {
       type = XSD_STRING;
     }
+    if(type === RDF_JSON_LITERAL) {
+      type = '@json';
+      rval['@value'] = JSON.parse(rval['@value']);
```

Comment:
@davidlehn -- should we do try/catch and wrap in a `JsonLdError` here?

Reply:
Wrapped.  Made up names and codes.  Not sure if that is ok or there should be errors for this case in the spec.